packages/e2e-tests/specs/experiments/multi-entity-saving.test.js

```diff
@@ -175,6 +175,11 @@ describe( 'Multi-entity save flow', () => {
 			await page.keyboard.press( 'Tab' );
 			await page.keyboard.press( 'Enter' );
 
+			// Wait to ensure template part is created/inserted before saving.
+			await new Promise( ( resolve ) => {
```

Comment:
Its there a way to replace this with `waitForSelector` or something like that. We avoid timeouts and things like that in tests because these are very fragile.

Reply:
Commented too soon, just saw your comment above. I'll think about it but yeah we'll have to find a more reliable way.

Reply:
Yeah 🤔 , the selector for the inserted template part doesn't seem distinguishable from other template parts that may be present.  I wonder if it would make sense to add the created template's slug as part of a classname here?